lizrd/support/misc.py

```diff
@@ -25,6 +25,18 @@ def count_parameters(model, args, VOCAB_SIZE):
     return model_n_params
 
 
+def count_moe_non_emb_active_params(args):
+    return (
+        args.dmodel**2
+        * (4 + 2 * (args.effective_dff_x if args.effective_dff_x is not None else 4))
+        * args.n_blocks
+    )
```

Comment:
Why not dmodel, effective_dff_x, n_blocks instead of args?

Reply:
What if dff is not 4 * dmodel? We should check that.

Reply:
Ok yup, I'll change to args.dff if args.effective_dff_x is none

Reply:
I really think passing these arguments here would add unnecessary, error-prone verbosity to function calls. This function is meant to use any parameters that are needed to calculate activated params, the caller does not have to know what's used under the hood.

Reply:
changed that